Changes in branch origin/3.0.x, between 952b803943689ec80cf3adb9824e0ba39951f526 and e4de342e0d053c85a9721fce6c734ad02f31913b
commit e4de342e0d053c85a9721fce6c734ad02f31913b
tree 57f3db6908d91d6a0a4880fe2984d59a2be58703
parent 952b803943689ec80cf3adb9824e0ba39951f526
author prachi <prachi@cloud.com> 1332797357 -0700
committer prachi <prachi@cloud.com> 1332797948 -0700

    Bug 14482 - When user tries to delete an acquired Ip address ,right after this is successful, UI calls listRemoteAccessVpns() which results in java.lang.NullPointerException.User is presented with "Internal error executing command" error
    
    Reviewed-By: Sheng
    
    Changes:
    - Added the null check for account object.

:100755 100755 a844e8c15f9a858e122ed1ee4065956647602b3c 988942040e256a394fa66c958cbe6eac13231a55 M	server/src/com/cloud/acl/DomainChecker.java
